lib/main.ts

```diff
@@ -130,13 +129,20 @@ export class Transpiler {
   /**
    * Transpiles the given files to Dart.
    * @param fileNames The input files.
-   * @param destination Location to write files to. Creates files next to their sources if absent.
+   * @param destination Location to write files to. Outputs file contents to stdout if absent.
    */
   transpile(fileNames: string[], destination?: string): void {
     if (this.options.basePath) {
       this.options.basePath = this.normalizeSlashes(path.resolve(this.options.basePath));
     }
-    fileNames = fileNames.map((f) => this.normalizeSlashes(f));
+    fileNames = fileNames.map((name: string) => {
+      const normalizedName = this.normalizeSlashes(name);
+      if (normalizedName.slice(0, 2) === './') {
```

Comment:
nit:
string.startsWith
instead of slice for this.
https://www.w3schools.com/jsref/jsref_startswith.asp